### PR TITLE
new conferences were added

### PR DESCRIPTION
**Pull request by sultankocaman**

### CODE REVIEW

**Review by ckenst — CHANGES_REQUESTED** [review]
@sultankocaman Can you re-order your additions, so they fit into the timeline? The conferences show based where they are in the list, so we need to add them properly to the timeline.

File: _data/current.yml [hunk 1]
```diff
@@ -266,3 +266,21 @@
   dates: "November 19-22, 2024"
   url: https://agiletestingdays.com/?utm_source=testingconferences
   status: Super Early Bird Pricing is Open until April 26, 2024
+  
+- name: The Premier Online Conference For Software Testing
+  location: online
+  dates: "May 21-22, 2024"
```
Comment (ckenst) [inline _data/current.yml:272]: Needs to be with the May Conferences

File: _data/current.yml [hunk 2]
```diff
@@ -266,3 +266,21 @@
...
+  url: https://live2test.com/
+  status: Super Early Bird Pricing is Open until end of March 2024
+  
+- name: Test İstanbul
+  location: online
+  dates: "October 10-11, 2024"
```
Comment (ckenst) [inline _data/current.yml:278]: Needs to be moved with the October conferences

File: _data/current.yml [hunk 3]
```diff
@@ -266,3 +266,21 @@
...
+  url: https://testistanbul.org/
+  status: Early Bird Pricing is Open Until May 31, 2024
+  
+- name: İstanbul Software Testing Conference
+  location: Istanbul, Turkey
+  dates: "May 23-24, 2024"
```
Comment (ckenst) [inline _data/current.yml:284]: Needs to be with the May Conferences

**Comment by sultankocaman** [comment]
> @sultankocaman Can you re-order your additions, so they fit into the timeline? The conferences show based where they are in the list, so we need to add them properly to the timeline.

I corrected the issues you mentioned in your comment. In this commit - >[Conferences sorted by date]

File: _data/current.yml (hunk 3 above)
Comment (sultankocaman) [reply _data/current.yml]: Fixed by the last commit - > [Conferences sorted by date]

File: _data/current.yml (hunk 2 above)
Comment (sultankocaman) [reply _data/current.yml]: Fixed by the last commit - > [Conferences sorted by date]

File: _data/current.yml (hunk 1 above)
Comment (sultankocaman) [reply _data/current.yml]: Fixed by the last commit - > [Conferences sorted by date]

**Review by ckenst — APPROVED** [review]

**Comment by ckenst** [comment]
@sultankocaman thanks for the changes!

**Pull request merged**